Sources/Onboarding/ProductNotifications/CookieNotificationBlockingConsentView.swift

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import DesignSystem
+import BraveShared
+import Growth
+import BraveUI
+
+public struct CookieNotificationBlockingConsentView: View {
+  public static let contentHeight = 400.0
+  public static let contentWidth = 344.0
+  private static let textPadding = 24.0
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  public var body: some View {
+    ScrollView {
+      VStack(alignment: .center, spacing: 48) {
+        VStack(alignment: .center, spacing: Self.textPadding) {
+          Image("cookie-consent-icon", bundle: .module)
+            .resizable()
+            .frame(width: 48, height: 48)
+          
+          Text(Strings.blockCookieConsentNoticesPopupTitle).font(.title)
+          Text(Strings.blockCookieConsentNoticesPopupDescription).font(.body)
+        }
+        .padding(Self.textPadding)
+        .foregroundColor(Color(UIColor.braveLabel))
+        .multilineTextAlignment(.center)
+        .fixedSize(horizontal: false, vertical: true)
+        
+        Button(Strings.blockCookieConsentNoticesDismissButtonTitle,
```

Comment:
In Figma there's "Stop blocking" button as well. Do we not add it?

Reply:
I understood that this is no longer needed. Perhaps I was mistaken.